src/azure-cli/azure/cli/command_modules/rdbms/_breaking_change.py

```diff
@@ -3,16 +3,13 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from azure.cli.core.breaking_change import register_command_group_deprecate, register_default_value_breaking_change, \
+from azure.cli.core.breaking_change import register_command_group_deprecate, \
     register_argument_deprecate, register_other_breaking_change, register_logic_breaking_change
 
 
 register_logic_breaking_change('postgres flexible-server create', 'Update default value of "--sku-name"',
                                detail='The default value will be changed from "Standard_D2s_v3" to a '
                                'supported sku based on regional capabilities.')
-register_default_value_breaking_change('postgres flexible-server create', '--version', '16', '17')
-register_default_value_breaking_change('postgres flexible-server create', '--create-default-database', 'Enabled',
-                                       'Disabled')
```

Comment:
While removing these upcoming breaking change announcement, you should mention these breaking changes in PR history so that customers can be aware that changes already happen